src/server_manager/scripts/get_electron_build_flags.mjs

```diff
@@ -33,24 +33,23 @@ export async function getElectronBuildFlags(platform, buildMode) {
       break;
     case 'windows':
       buildFlags = [
-        '--win', '--ia32',
+        '--win',
+        '--ia32',
         '--config.win.icon=icons/win/icon.ico',
         '--config.win.sign=src/server_manager/electron_app/windows/electron_builder_signing_plugin.cjs',
-        ...buildFlags];
+        ...buildFlags,
+      ];
       break;
     case 'macos':
       buildFlags = ['--mac', '--config.mac.icon=icons/mac/icon.icns', ...buildFlags];
   }
 
   if (buildMode === 'release') {
-    // Publishing is disabled, updates are pulled from AWS. We use the generic provider instead of the S3
-    // provider since the S3 provider uses "virtual-hosted style" URLs (my-bucket.s3.amazonaws.com)
-    // which can be blocked by DNS or SNI without taking down other buckets.
     buildFlags = [
       ...buildFlags,
       '--config.generateUpdatesFilesForAllChannels=true',
-      '--config.publish.provider=generic',
-      `--config.publish.url=${process.env.RELEASES_REPOSITORY}`,
+      '--config.publish.provider=github',
```

Comment:
Why is this changing?

Reply:
I think this has to be reverted to generic, since we are not using github releases: https://www.electron.build/configuration/publish.html

Reply:
seems this was a copy paste error from the S3 code in the client - https://www.electron.build/configuration/publish.html provider should be `github`?

Reply:
mm i see np

Reply:
We are specifying the URL. It should be generic. That's what we were using before: https://github.com/Jigsaw-Code/outline-server/blob/a19e0a0090ccd570697223b4f89ad62f82025a6d/src/server_manager/electron_app/release_windows_action.sh

Reply:
ah seems i didn't read enough. reverting.